src/winetricks

```diff
@@ -11971,6 +11971,60 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    if [ "$W_ARCH" == "win64" ]; then
+        w_die "Installation of Office 2013 Pro will not succeed on a 64bit wine prefix"
+    fi
+
+    if ! command -v winbindd > /dev/null; then
+        w_warn "winbind is required"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    if ! test -f "$W_CACHE"/office2013pro/OFFICE15.iso; then
+        w_download http://care.dlservice.microsoft.com/dl/download/2/9/C/29CC45EF-4CDA-4710-9FB3-1489786570A1/OfficeProfessionalPlus_x86_en-us.img 236f8faae3f979ec72592a63784bba2f0d614916350c44631221b88ae9dae206 "OFFICE15.iso"
+    fi
+
+    w_set_winver win7
+
+    w_call corefonts
+    w_call msxml6
+    w_call riched20
```

Comment:
according to the latest test result by Yossi Synett on [winehq](https://appdb.winehq.org/objectManager.php?bShowAll=true&bIsQueue=false&bIsRejected=false&sClass=version&sTitle=&sReturnTo=&iId=26323) `dotnet40` and `msxml6` are not necessary. Only `riched20` is needed.

another report says:
> msxml6)...without this, it will error out during initial setup at around 68% with an XSD parse error
but that is definitely wrong with `wine 2.0` and `wine 3.0`. I think dropping msxml6 is fine.

Reply:
I think dropping `msxml6` is fine and we don't need `dotnet20 msftedit riched30 vb6run`. Do we agree here?

Reply:
linked the winehq bugreports and dropped all other

https://github.com/Winetricks/winetricks/pull/939/commits/b507f2f31b41ef6a8dbd4cb10fb8ce9f5d9bf6ef

Reply:
vb6run is needed for vba support.

Reply:
it is not needed for installation. I would consider it an optional feature.